rinja/src/filters/builtin.rs

```diff
@@ -230,6 +448,26 @@ impl<W: fmt::Write> fmt::Write for TruncateWriter<W> {
 }
 
 /// Indent lines with `width` spaces
+///
+/// ```
+/// /// ```
+/// # #[cfg(feature = "code-in-doc")] {
+/// # use rinja::Template;
+/// /// ```jinja
+/// /// <div>{{ example|indent(4) }}</div>
+/// /// ```
+/// #[derive(Template)]
+/// #[template(ext = "html", in_doc = true)]
+/// struct Example<'a> {
+///     example: &'a str,
+/// }
+///
+/// assert_eq!(
+///     Example { example: "hello\nfoo\nbar" }.to_string(),
+///     "<div>hello\n    foo\n    bar</div>"
```

Comment:
I'm surprised that the first line is not indented.

Reply:
Ok, from [jinja docs](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.indent), by default the first line and blank lines are not indented. We should maybe support to add more options.

Reply:
Opened https://github.com/rinja-rs/rinja/issues/193 for keeping track of that.